src/css/components/_volume.scss

```diff
@@ -41,27 +41,34 @@
   left: 0;
 
   background-color: $primary-text;
+
+  @extend .vjs-icon-circle;
+
+  &:before {
+    position: absolute;
+  }
 }
 
-.video-js .vjs-slider-vertical .vjs-volume-level { width: 0.3em; }
-.video-js .vjs-slider-horizontal .vjs-volume-level { height: 0.3em; }
+.video-js .vjs-slider-vertical .vjs-volume-level {
+  width: 0.3em;
+
+  &:before {
```

Comment:
It might be helpful to add some comments around these to make it clear that it's the handle being affected here. It's not obvious at first glance.